climpred/bootstrap.py

```diff
@@ -316,13 +414,10 @@ def bootstrap_compute(
         shuffle_dim = 'init'
     else:
         raise ValueError('Shuffle either `member` or `init`; not', dim)
+
     # resample with replacement
-    # DoTo: parallelize loop
-    for _ in tqdm(range(bootstrap), desc='bootstrapping iteration'):
```

Comment:
tqdm bar doesnt make sense here when lazily results because it immediately runs through but computation is later

Reply:
Maybe find a way to only use `tqdm` if the datasets are not `dask` datasets? Then if they are the users can either use the `dask_jobqueue` dashboard if on multiple nodes or can use the dask progressbar if they are on a single node.

Reply:
Never used the dask progress bars.  But we can also just leave the tqdm in always. Doesn’t hurt

Reply:
Here's the `dask` progress bars: https://stackoverflow.com/questions/49039750/how-to-see-progress-of-dask-compute-task.

This is something that the user can just do in their notebook and they get if they are using `dask`. `tqdm` on the other hand we have to do internally for it to work. So if it's non-dask I suggest we wrap it in `tqdm` and if it's `dask` wrap it in nothing so they don't get a progress bar that's misleading.

I'm not sure how to do that elegantly though,